app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -125,7 +126,8 @@ export default class SampleDetails extends React.Component {
 
     this.onUIStoreChange = this.onUIStoreChange.bind(this);
     this.clipboard = new Clipboard('.clipboardBtn');
-    this.addManualCas = this.addManualCas.bind(this);
+    this.checkCasValidation = this.checkCasValidation.bind(this);
+    this.addHypenToCas = this.addHypenToCas.bind(this);
```

Comment:
Minor nitpick: in case the "-" symbol is meant, it should be "hyphen", not "hypen".

Reply:
fixed